src/PeriodEnd/LevyPayments/DeployLevyPayments/sql/Migration Scripts/1_LevyPayments.deds.Add_Indexes.sql

```diff
@@ -1,4 +1,10 @@
-
```

Comment:
IF EXISTS - DROP INDEX  check required ?

Reply:
Done, although as mentioned on Slack, we might want to change the check pattern we use for index creation to...
"if not exists, create"
but if all the indexes are on the transient db, rather than deds, it's not an issue